parser/prism/Translator.cc

```diff
@@ -1164,6 +1083,10 @@ unique_ptr<parser::Node> Translator::translate(const Node &node) {
     return translate(node.get_raw_node_pointer());
 }
 
+core::LocOffsets Translator::translate(pm_location_t loc) {
```

Comment:
Personally, I don't think this overload of the `translate` method is very nice, since at call sites it reads like it is recursing into translating some child node, but it is not acting on any nodes at all. Some other name like `translateLoc` would be my preference.